packaging/examples/security/keycloak-authorization/README.md

```diff
@@ -14,5 +14,8 @@ Full instructions for the example are available in the [Strimzi Documentation](h
 - [kafka-ephemeral-oauth-single-keycloak-authz.yaml](./kafka-ephemeral-oauth-single-keycloak-authz.yaml)
   - The Kafka CR that defines a single-node Kafka cluster with `oauth` authentication and `keycloak` authorization,
     using the `kafka-authz` realm. See [full example instructions](https://strimzi.io/docs/operators/in-development/configuring.html#proc-oauth-authorization-keycloak-example_str) for proper preparation and deployment.
+- [kafka-ephemeral-oauth-single-keycloak-authz-metrics.yaml](./kafka-ephemeral-oauth-single-keycloak-authz-metrics.yaml)
```

Comment:
Do we want to say something about this in this section: https://strimzi.io/docs/operators/in-development/deploying.html#example_files_provided_with_strimzi

Reply:
You mean add something like:

The `keycloak-authorization` directory also contains an example with enabled metrics for `oauth` authentication and `keycloak` authorization.